src/test/java/com/dropbox/core/oauth/DbxRefreshTest.java

```diff
@@ -27,15 +27,16 @@
 import static org.mockito.Mockito.when;
 import static org.testng.Assert.fail;
 
-public class DbxRefershTest extends DbxOAuthTestBase {
+public class DbxRefreshTest extends DbxOAuthTestBase {
     private static final DbxRequestConfig CONFIG = DbxRequestConfig.newBuilder("DbxWebAuthTest/1.0")
         .withUserLocaleFrom(Locale.UK)
         .build();
     private static final DbxAppInfo APP = new DbxAppInfo("test-key", "test-secret");
     private static final String EXPIRED_TOKEN = "expired_token";
     private static final String REFRESH_TOKEN = "refresh____token";
     private static final String NEW_TOKEN = "new_token";
-    private static final long EXPIRES_IN = 14400;
+    private static final long EXPIRES_IN_SECONDS = 14400L;
+    private static final long OFFSET_BETWEEN_CALLS_IN_MS = 1000L;
```

Comment:
There is no way to inject the issuedTime into the client therefore I added an offset of 1 second.